app/models/package.rb

```diff
@@ -516,7 +516,11 @@ def is_singleton_package?
   end
 
   def update_location_quantity(total_quantity, location_id)
-    packages_locations.where(location_id: location_id).first.update_column(:quantity, total_quantity)
+    get_packages_location_with_location_id(location_id).update_quantity(total_quantity)
+  end
+
+  def get_packages_location_with_location_id(location_id)
```

Comment:
Do we use the function `get_packages_location_with_location_id` anywhere else? If we're just calling it from the line above then I think it's not necessary to turn it into a one line function. It makes it hard to trace the functionality and flow when reading through the code.

I think in general, I'd prefer not to have one-line functions unless they are called from at least 2 separate places... I know it's a style thing but I think it's a trade off of readability and functionality.